setup.cfg

```diff
@@ -1,6 +1,6 @@
 [pycodestyle]
 max-line-length=120
-ignore=E261,E302,E305
+ignore=E261,E302,E305,W503
```

Comment:
Should probably belong in a separate commit with an explanation of what this warning is